ArduPlane/fence.cpp

```diff
@@ -98,6 +87,21 @@ void Plane::fence_check()
                     guided_WP_loc.lat = home.lat;
                     guided_WP_loc.lng = home.lng;
                 }
+
+                float fence_alt_max, fence_alt_min;
```

Comment:
This should probably have been made into a `const float fence_breach_rtl_alt() const` method.